.devcontainer/tests/Dockerfile

```diff
@@ -4,3 +4,11 @@ FROM mcr.microsoft.com/devcontainers/${IMAGE}
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install --no-install-recommends postgresql-client \
      && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+
+WORKDIR /workspace
+
+COPY README.md LICENSE.txt ../../uv.lock pyproject.toml setup.cfg .python-version .
```

Comment:
I'm curious why the partial copy here / how you choose what to copy? Going forward, when would this need to be adjusted with additional files?

Reply:
I only want to copy the smallest set of files required to install requirements, so that docker can cache layers more effectively.

If we copy the entire repo inside at this point, any change to any file will mean docker wants to rebuild the following layers. In this case it's probably not a huge concern, but would be technically less efficient.

Reply:
Yep this is a mistake my side, leftover from when I wasn't doing the entire `flask_admin` copy. Good spot thanks